### PR TITLE
cleanup ISSUE_TEMPLATE (fixes #1337)

### LINKED ISSUE #1337: Improve consistency of issue template
The current issue template has some style inconsistencies: some headings end in colons, others don't; capitalization of headings is a mix of Sentence case and Title Case; some spaces are missing, e.g. before parenthesis, etc.

It should be edited to be at least internally consistent.

### PR DESCRIPTION
**Pull request by waldyrious**
- Harmonize capitalization in headings and elsewhere
- Remove colons in headings.
  (to avoid encouraging addition of information inline rather than below)
- Other minor tweaks (spacing, spelling, etc.)

### CODE REVIEW

**Comment by srirambv** [comment]
Please log an issue and use that issue in closing keyword on the PR.

**Comment by waldyrious** [comment]
Oh wow, I got the lucky issue #1337! Anyway, done. 👍

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -3,38 +3,37 @@
...
 
-## Steps to Reproduce <!--Please add a series of steps to reproduce the issue-->
+## Steps to reproduce <!-- Please add a series of steps to reproduce the issue -->
 
    1. 
    2. 
    3. 
 
-## Actual result: <!--Please add screenshots if needed-->
+## Actual result <!-- Please add screenshots if needed -->
```
Comment (srirambv) [inline ISSUE_TEMPLATE.md]: Would be good to have the extra line after this.

Comment (waldyrious) [reply ISSUE_TEMPLATE.md]: You mean add, right? Will do.

Comment (srirambv) [reply ISSUE_TEMPLATE.md]: Looks good. Thanks

**Pull request merged**